src/UI/examples/Button/Standard/with_js_binding.php

```diff
@@ -0,0 +1,14 @@
+<?php
+function with_js_binding() {
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    return $renderer->render(
+		$f->button()->standard("Goto ILIAS", "#")
```

Comment:
Absolutely valid point, as long as the "action" is a stringy link. I would like to defer that to the introduction of a proper Action-abstraction, ok? Until then, a "" will imo do.